src/vstart.sh

```diff
@@ -542,6 +544,13 @@ start_mon() {
 			--cap mgr 'allow *' \
 			"$keyring_fn"
 
+		prun $SUDO "$CEPH_BIN/ceph-authtool" --gen-key --name=client.rgw --set-uid=0 \
+		    --cap mon 'allow *' \
+		    --cap osd 'allow *' \
+		    --cap mds 'allow *' \
+		    --cap mgr 'allow *' \
+		    "$keyring_fn"
```

Comment:
ok, I'm fine with the keyring actually it does allow us to have seperate debug levels and config sections